packages/widgets/src/lib/components/InferenceWidget/widgets/ConversationalWidget/ConversationalWidget.svelte

```diff
@@ -66,13 +73,28 @@
 			updateUrl({ text: trimmedText });
 		}
 
-		const requestBody = {
-			inputs: {
-				generated_responses: conversation.generated_responses,
-				past_user_inputs: conversation.past_user_inputs,
-				text: trimmedText,
-			},
-		};
+		// const chat = [
+		// 	{ role: "user", content: "Hello, how are you?" },
+		// 	{ role: "assistant", content: "I'm doing great. How can I help you today?" },
+		// 	{ role: "user", content: "I'd like to show off how chat templating works!" },
+		// ];
+		const chat: Message[] = []; // TODO
+		const chatTemplate = model.config?.tokenizer?.chat_template;
+		if (chatTemplate === undefined) {
+			outputJson = "";
+			output = [];
+			error = "No chat template found in tokenizer config";
+			return;
+		}
+
+		const template = new Template(chatTemplate);
+		const chatText = template.render({
+			messages: chat,
+			bos_token: model.config?.tokenizer?.bos_token,
+			eos_token: model.config?.tokenizer?.eos_token,
+		});
```

Comment:
Also, I think `add_generation_prompt` should be set here, right? cc @xenova (unless it's set by default in the JS implementation)

Reply:
Yes that's correct, we should set it here too. Transformers.js sets the default value, but since we are using `@huggingface/jinja` directly, we need to specify all variables.